README.md

```diff
@@ -1,4 +1,17 @@
 # waiter
 
 ## Description
-A server that establishes a connection with the client, recieves their message and echos it back to them
\ No newline at end of file
+A server that establishes a connection with the client, receives their message and echos it back to them
+
+## Prerequisites
+- gradle build
```

Comment:
Does this assume I already have gradle installed?

Reply:
That is a good point, It does, I should put that in the prereqs to install it, woops 😮

Reply:
Does this also assume I have Java installed? If so, which version of Java?

Reply:
Another good point, I missed them all! 😵‍💫 The version is Java 18.0.1.1

Reply:
I added both of these install steps to the prereqs of the README!

Reply:
Check for resolved in [this commit](https://github.com/jonmanahan/waiter/pull/2/commits/e53fdbc1eec22480eae57e79838d0f1186c16f90), if it resolves the issues please hit resolve conversation if possible, thank you all a ton!

Reply:
I was able to run the project and tests with the following:

`openjdk-18.0.1.1` (installed via `asdf`)
`gradle-7.4.2` (installed via `brew`)

Reply:
Should I put the gradle version in the README like I did the Java?

Reply:
I'll leave that up to you. The Java version is more important. You could specify that the application is known to work with X version of Gradle if you want.